queries.sql

```diff
@@ -0,0 +1,119 @@
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS channel;
+DROP TABLE IF EXISTS user;
+DROP TABLE IF EXISTS message;
+DROP TABLE IF EXISTS user_channel;
+
+CREATE TABLE organization (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128)
+);
+
+CREATE TABLE channel (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128),
+  org_id INTEGER REFERENCES organization(id)
+);
+
+CREATE TABLE user (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  name VARCHAR(128)
+);
+
+CREATE TABLE message (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  post_time TIMESTAMP NOT NULL DEFAULT(DATETIME()),
+  content TEXT NOT NULL,
+  user_id INTEGER REFERENCES user(id),
+  channel_id INTEGER REFERENCES channel(id)
+);
+
+CREATE TABLE user_channel (
+  user_id INTEGER REFERENCES user(id),
+  channel_id INTEGER REFERENCES channel(id)
+);
+
+-- Organization
+INSERT INTO organization (name) VALUES ("Lambda School");
+
+-- Users
+INSERT INTO user (name) VALUES ("Alice");
+INSERT INTO user (name) VALUES ("Bob");
```

Comment:
It is also possible to make that insertion in a single query with coma's for values.